### ISSUE TITLE
Having problems with the Context menu on mobile

### ISSUE THREAD

**Issue by pikatwinky** [zsviczian/obsidian-excalidraw-plugin#742]
I know maybe this issue belongs to the Excalidraw.com repo (i'll rise it here and there as well, just in case), but i'd like to know if there is a simple workaround to this issue (maybe). 

When i use excalidraw plugin on mobile i have problems regularly with the context menu, i want to access to the **lock button** located almost to the botton of the bar, but in my devices the bar most of the times overflows to the bottom of the screen, only in certain occasions the context menu is shown properly like in the next video (but to achive that i have to try many times)   

https://user-images.githubusercontent.com/31860235/184465241-44e9f04f-7260-4a22-9817-1fd49fe34f97.mp4

My own workaround is to displace the sketch all to the top to being able to reach the lock button, however i have another device with a smaller screen in which even this approach doesn't work (the size of the screen from the video was 6.53" ), it also consumes time doing this each time i want to lock something

I know there were fixes related to this, but i'm a bit confused if they doesn't apply to this case as well (refering to https://github.com/excalidraw/excalidraw/issues/3732 and https://github.com/excalidraw/excalidraw/pull/4030)

In fact due to this i think the Zsolt propose of a radial context menu could be awesome https://github.com/excalidraw/excalidraw/issues/4986

**Comment by zsviczian** [comment]
since a few weeks the context menu is scrollable. I just tried this, and indeed it works. Try sliding the menu with your finger.

**Comment by pikatwinky** [comment]
@zsviczian, Oh, in the video it is not appreciated but i was scrolling all the time and the bar didn't move at all. I don't know if maybe i'm doing something wrong. 


(I'm sorry i tried to upload the video evidence but it's too large)

I slide my finger....
![Screenshot_2022-08-13-11-07-33-058_com miui gallery](https://user-images.githubusercontent.com/31860235/184502125-8b7fdcd5-e433-4cfb-842e-8497d029d6ae.jpg)

and the bar doesn't move at all...
![Screenshot_2022-08-13-11-07-48-593_com miui gallery](https://user-images.githubusercontent.com/31860235/184502131-f4c494bc-d64a-44a4-a8a2-13028225c3f2.jpg)
Where my finger ends is the low border of the screen, the **lock** button is yet further, and scrolling is not happening for some odd reason

**Comment by zsviczian** [comment]
and you are using the latest version of the plugin and of Obsidian mobile?
What device are you using? Which version of the app and the plugin are you using?
Are you using a custom Obsidian theme? Can you test without the custom theme and custom CSS snippets (if any)?

**Comment by pikatwinky** [comment]
Yes, i think I have the latest realeases of obsidian for android (0.15.6) and the plugin (1.7.12).
I don't use custom CSS snippets, but i was using a custom theme called "Wasp", however after returning to the default theme the problems are still present.
I started recently using the Andy's Sliding Panes plugin, maybe the problem could be in part due to this, but i'd like to describe briefly what i've experienced
  
I have two mobiles Android, in both of them i have this problem but slightly different
* Xiaomi Redmi Note 8 Pro (screen 6.53"): The scrolling feature is not present at all (with or without the custom theme and the sliding panes plugin), i can't see the effect of scrolling and instead the menu always overflows the screen when i try to select a figure near the bottom border of the Excalidraw canvas. The behaviour is as depicted in the images above
* Motorola E4 plus (screen  5.5”) : Using the sliding panes, the behaviour is like in the video, but when i remove the Sliding Panes plugin, the scrolling does occur at some degree, however it doesn't scroll through the entire menu (not reaching bottom buttons like the lock one). I mean, when i try to access the context menu of figures near the bottom border of the screen i can scroll but not completly (only partially) 
What i noticed is what when i start obsidian for first time and try to reach a figure from the bottom border, only the first time the context menu is shown properly, 
![Screenshot_20220813-122122](https://user-images.githubusercontent.com/31860235/184504527-32980498-4fea-48fc-bb90-e5bb6dfcfdf6.png)
Here we can see the whole context menu displayed, 
but after that if i click again the same figure near the same point, not anymore, the context menu overflows to the bottom
![Screenshot_20220813-122129](https://user-images.githubusercontent.com/31860235/184504529-07a3c938-8224-4220-acf4-51a90bb6b162.png)

and i have no move the whole drawing to the top. This holds with and without the Sliding Panes Plugin

**Comment by zsviczian** [comment]
and on both devices you can reproduce the issue on excalidraw.com?
with which browser? The default browser by the vendor or the chome browser?

**Comment by pikatwinky** [comment]
Now you mention that, i was using only chrome based browsers when i tested this on excalidraw.com. 
I tried again using Firefox and the vendor default browser (Mi browser) and it works nicely on both phones.

**Comment by zsviczian** [comment]
so you can try changing webview on your phone... I did this few years ago, you don't need a rooted phone for this, but I don't quite remember how to do it.

I think the steps are something like this:
- download chrome browser from play store
- in settings/app enable to show system apps
- select webview from the list and click disable
- turn on developer options (google for help... I think you need to click About Phone 10x times)
- in developer options search for the setting WebView Implementation
- change it to chrome

I think this will fix your issue...

**Comment by pikatwinky** [comment]
I'm sorry but it didn't work, in fact changing to chrome WebView freezes the obsidian app when opening. I had to return to the Android WebView again. 
I have a whole set of evidence again but instead that, i want ask if i can access the lock button functionality through an script maybe or in another way I don't know yet? After all that is my real issue, reach that button functionality (or some others from the context menu which are not reachable)
